tests/bluetooth/tester/qemu.conf

```diff
@@ -18,5 +18,17 @@ CONFIG_BT_DEBUG_ATT=y
 CONFIG_BT_DEBUG_GATT=y
 CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_PROV=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_GATT_PROXY=y
+CONFIG_BT_MESH_SUBNET_COUNT=5
+CONFIG_BT_MESH_APP_KEY_COUNT=5
+CONFIG_BT_MESH_MODEL_GROUP_COUNT=5
+CONFIG_BT_MESH_LABEL_COUNT=5
```

Comment:
Same here. You're unnecessarily bloating the resulting image, so that it may not be testable on some lower memory boards.

Reply:
ok, I'll use default values then